stock_request/views/stock_request_views.xml

```diff
@@ -11,22 +11,27 @@
                 decoration-bf="message_needaction==True"
             >
                 <field name="message_needaction" invisible="1" />
+                <field name="allow_virtual_location" invisible="1" />
                 <field name="name" />
                 <field name="warehouse_id" groups="stock.group_stock_multi_locations" />
                 <field name="location_id" groups="stock.group_stock_multi_locations" />
-                <field name="allow_virtual_location" invisible="1" />
                 <field
                     name="route_id"
                     options="{'no_create': True}"
                     groups="stock.group_stock_multi_locations"
                 />
                 <field name="product_id" />
-                <field name="allowed_uom_categ_id" invisible="1" />
                 <field
                     name="product_uom_id"
                     options="{'no_open': True, 'no_create': True}"
                     groups="uom.group_uom"
                 />
+                <field
+                    name="product_uom_id"
+                    readonly="True"
```

Comment:
I understand what you mean, but it currently has force_save=1 set. If we add the field as invisible (without groups) and force_save, it could be incompatible with the same field added just before with groups? 

IMO, I think it should be invisible=1, force_save=1 and with negative groups.

Reply:
It's not incompatible. All occurrences of the same field area synced. This field should be invisible, without force save (unless on the definition in python there's readonly), and without groups.